From 9b38fda6feda68f95754d5b8932b1a69471df960 Mon Sep 17 00:00:00 2001
From: Hiltjo Posthuma <hiltjo@codemadness.org>
Date: Thu, 11 Jun 2020 18:45:33 +0200
Subject: [PATCH 6/6] Fix memory leaks in drw

Synced from dwm.
Patch by Alex Flierl <shad0w73@freenet.de>, thanks.
---
 drw.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drw.c b/drw.c
index 8fd1ca4..4cdbcbe 100644
--- a/drw.c
+++ b/drw.c
@@ -95,6 +95,7 @@ drw_free(Drw *drw)
 {
 	XFreePixmap(drw->dpy, drw->drawable);
 	XFreeGC(drw->dpy, drw->gc);
+	drw_fontset_free(drw->fonts);
 	free(drw);
 }
 
-- 
2.27.0

